### PR TITLE
Simplify parent and child into strings instead of full-blown messages

### PR DESCRIPTION
**Pull request by dmathieu**
This is part of #47.

### CODE REVIEW

File: model/proto/apmevent.proto [hunk 1]
```diff
@@ -76,15 +74,15 @@ message APMEvent {
   Processor processor = 19;
   HTTP http = 20;
   UserAgent user_agent = 21;
-  Parent parent = 22;
+  string parent = 22;
```
Comment (marclop) [inline model/proto/apmevent.proto:77]: I think this should be named `parent_id` not just parent?
```suggestion
  string parent_id = 22;
```

Comment (dmathieu) [reply model/proto/apmevent.proto]: Makes sense.
I have also renamed `child` to `childs`.

Comment (marclop) [reply model/proto/apmevent.proto]: shouldn't it be `children`?

Comment (dmathieu) [reply model/proto/apmevent.proto]: It was in the short form, so I didn't extend it. But I can do so.

**Review by marclop — APPROVED** [review]

**Pull request merged**